sanfrancisco-buildings/index.html

```diff
@@ -6,7 +6,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
     <title>San Francisco buildings</title>
-    <link rel="stylesheet" href="https://js.arcgis.com/4.24/esri/css/main.css" />
+    <link rel="stylesheet" href="https://js.arcgis.com/4.25/esri/css/main.css" />
```

Comment:
I couldn't update SF buildings all the way to 4.26 because of some weird editing issue, see issue https://github.com/RalucaNicola/JSAPI_demos/issues/20. Didn't have time to check if this is because of your app or because of a regression in the API. I'll report this to the experience team.